packages/cacti-plugin-copm-fabric/src/test/typescript/integration/test-copm-pledge-claim.test.ts

```diff
@@ -0,0 +1,255 @@
+import { AddressInfo } from "node:net";
+import http from "node:http";
+
+import "jest-extended";
+import { v4 as uuidV4 } from "uuid";
+import { createPromiseClient, PromiseClient } from "@connectrpc/connect";
+import { createConnectTransport } from "@connectrpc/connect-node";
+import { PluginRegistry } from "@hyperledger/cactus-core";
+import {
+  LogLevelDesc,
+  Logger,
+  LoggerProvider,
+  Servers,
+} from "@hyperledger/cactus-common";
+import { ApiServer } from "@hyperledger/cactus-cmd-api-server";
+import { AuthorizationProtocol } from "@hyperledger/cactus-cmd-api-server";
+import { ConfigService } from "@hyperledger/cactus-cmd-api-server";
+import {
+  DefaultService,
+  ClaimPledgedAssetV1Request,
+  PledgeAssetV1Request,
+  AssetAccountV1PB,
+  DLAccount,
+} from "@hyperledger/cacti-copm-core";
+import { PluginCopmFabric } from "../../../main/typescript/plugin-copm-fabric";
+import { CopmWeaverFabricTestnet } from "../lib/copm-weaver-fabric-testnet";
+import { TestAssetManager } from "../lib/test-asset-manager";
+import * as path from "path";
+import * as dotenv from "dotenv";
+dotenv.config({ path: path.resolve(__dirname, "../../.env") });
+
+const logLevel: LogLevelDesc = "DEBUG";
+const log: Logger = LoggerProvider.getOrCreate({
+  label: "plugin-copm-crpc-server-test",
+  level: logLevel,
+});
+
+describe("PluginCopmFabric", () => {
```

Comment:
@jenniferlianne I tried to run `yarn configure` and then this test case but it crashed with the stack trace below. Could you please take a look? In general the test cases should be operational after the configure script has executed so I'm guessing maybe some part of the build process is not integrated into `configure` but that's just an idea.

Here's the exact command I ran and the log it produced:

```sh
$ yarn test:jest:all packages/cacti-plugin-copm-fabric/src/test/typescript/integration/test-copm-pledge-claim.test.ts
jest-haste-map: duplicate manual mock found: ConfigUtil
  The following files share their name; please delete one of them:
    * <rootDir>/examples/cactus-common-example-server/src/main/typescript/routing-interface/util/__mocks__/ConfigUtil.ts
    * <rootDir>/examples/cactus-common-example-server/dist/lib/main/typescript/routing-interface/util/__mocks__/ConfigUtil.js

jest-haste-map: duplicate manual mock found: ConfigUtil.d
  The following files share their name; please delete one of them:
    * <rootDir>/examples/cactus-common-example-server/dist/lib/main/typescript/routing-interface/util/__mocks__/ConfigUtil.d.ts
    * <rootDir>/packages/cactus-cmd-socketio-server/dist/lib/main/typescript/routing-interface/util/__mocks__/ConfigUtil.d.ts

jest-haste-map: duplicate manual mock found: ConfigUtil
  The following files share their name; please delete one of them:
    * <rootDir>/examples/cactus-common-example-server/dist/lib/main/typescript/routing-interface/util/__mocks__/ConfigUtil.js
    * <rootDir>/packages/cactus-cmd-socketio-server/dist/lib/main/typescript/routing-interface/util/__mocks__/ConfigUtil.js

 FAIL  packages/cacti-plugin-copm-fabric/src/test/typescript/integration/test-copm-pledge-claim.test.ts
  ● Test suite failed to run

    Configuration error:
    
    Could not locate module ./build/src/Relay.js mapped as:
    $1.
    
    Please check your configuration for these entries:
    {
      "moduleNameMapper": {
        "/^(\.\.?\/.+)\.jsx?$/": "$1"
      },
      "resolver": undefined
    }

      13 | // likely due to that module containing multiple classes
      14 |
    > 15 | module.exports.RelayHelper = require("./build/src/Relay.js");
         |                              ^
      16 | module.exports.InteroperableHelper = require("./build/src/InteroperableHelper.js");
      17 | module.exports.AssetManager = require("./build/src/AssetManager.js");
      18 | module.exports.SatpAssetManager = require("./build/src/SatpAssetManager.js");

      at createNoMappedModuleFoundError (node_modules/jest-resolve/build/resolver.js:759:17)
      at Object.<anonymous> (weaver/sdks/fabric/interoperation-node-sdk/index.js:15:30)
      at Object.<anonymous> (packages/cacti-copm-core/src/main/typescript/validators/common.ts:5:1)
      at Object.<anonymous> (packages/cacti-copm-core/src/main/typescript/validators/validate_claim_locked_asset_request.ts:5:1)
      at Object.<anonymous> (packages/cacti-copm-core/src/main/typescript/validators.ts:1:1)
      at Object.<anonymous> (packages/cacti-copm-core/src/main/typescript/public-api.ts:24:1)
      at Object.<anonymous> (packages/cacti-copm-core/src/main/typescript/index.ts:1:1)
      at Object.<anonymous> (packages/cacti-plugin-copm-fabric/src/test/typescript/integration/test-copm-pledge-claim.test.ts:18:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        5.807 s
Ran all test suites matching /packages\/cacti-plugin-copm-fabric\/src\/test\/typescript\/integration\/test-copm-pledge-claim.test.ts/i.
```

Reply:
More info: I have not tested the other test cases but if they suffer from the same problem then I would also like to request those to be addressed as well (if the solution is somehow unique to each test case for whatever reason)

Reply:
These tests need a weaver network to run.  Following the weaver pattern, the network can either run 'lock' (asset exchange) or 'pledge' (asset transfer) commands.  To start a network, in the fabric directory type 
`make setup; make pledge-network`
Then wait 10 min.  More here: https://github.com/jenniferlianne/cacti/tree/copm_draft/packages/cacti-plugin-copm-fabric

Reply:
@jenniferlianne 

> These tests need a weaver network to run

The pattern I'm trying to encourage is that the tests are responsible for setting up and tearing down the testing infrastructure that they need so that the test itself can be executed with a single command. Having additional manual steps to run the test case can sink developer productivity like a bag of bricks.

With that said, I know you've worked a lot on the infrastructure already and this PR so the right trade-off seems like to let this go and keep track of it in a separate issue which I opened here for future reference: https://github.com/hyperledger/cacti/issues/3556